apps/web/lib/team/[slug]/getServerSideProps.tsx

```diff
@@ -175,6 +178,28 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     } as const;
   }
 
+  const organizationSettings = isOrgContext
+    ? team?.isOrganization
+      ? team?.organizationSettings
+      : team?.parent?.organizationSettings
+    : null;
+  let allowSEOIndexing = false;
+
+  if (!!organizationSettings) {
+    allowSEOIndexing = organizationSettings.allowSEOIndexing;
+
+    if (organizationSettings.disableOrgSubdomainURL) {
```

Comment:
Because we are not actually disabling orgSubDomain URL. We are just redirecting the home page. I think we should rename it to something like 
`orgProfileRedirectsToVerifiedDomain`. We call i.cal.com page to be organization profile page that lists all teams of that organizations and other details.

So, it reads like this `organizationSettings.orgProfileRedirectsToVerifiedDomain`

Reply:
I think we should update the copy(text) as well in the setting to better reflect it.

New text could be something like this.

'Disable public organization profile and redirect '
'Redirects deel.cal.com to deel.com'

cc @PeerRich. It is not a blocker, we can fix the text after the PR merge as well.

Reply:
ok will rename var accordingly and update the description